src/components/modals/RedemptionInitiatedModal.vue

```diff
@@ -0,0 +1,153 @@
+<script setup lang="ts">
+import { CheckCircleIcon } from '@heroicons/vue/24/outline'
+import BaseButton from '@uicommon/components/BaseButton.vue'
+import BaseModal from '@uicommon/components/BaseModal.vue'
+import { useModals } from '@uicommon/composables/useModals'
+import { formatCurrency } from '@uicommon/utils/numbers'
+import { useElementHover } from '@vueuse/core'
+import dayjs from 'dayjs'
+import { computed, ref, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import RedemptionInitiatedModalItem from './RedemptionInitiatedModalItem.vue'
+import { EModal, kDefaultDateTimeFormat, kMsInSecond } from '@/constants/ui'
+
+const { t } = useI18n()
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
+const { visibleModal, hideModal, modalProps } = useModals()
+
+const isPriceVisible = ref(false)
+
+const formattedRedemptionAmount = computed((): number | string => {
+  const rawRedemptionAmount: string = modalProps.value.redemptionAmount
+  return rawRedemptionAmount && Number(rawRedemptionAmount)
+    ? formatCurrency(Number(rawRedemptionAmount))
+    : 0
+})
+
+const formattedHfAmount = computed((): number | string => {
+  const rawHfAmount: string = modalProps.value.hfAmount
+  return rawHfAmount && Number(rawHfAmount) ? formatCurrency(Number(rawHfAmount)) : 0
+})
+
+const formattedDate = computed((): string =>
+  dayjs(new Date(modalProps.value.date / kMsInSecond)).format(kDefaultDateTimeFormat)
+)
+
+function showTransactionPrice(state: boolean): void {
+  if (formattedRedemptionAmount.value !== '---') {
+    isPriceVisible.value = state
+  }
+}
+
+// Hover effect
+const el = ref()
+const isHovered = useElementHover(el, { delayEnter: 300, delayLeave: 0 })
+
+watch(isHovered, (value) => {
+  showTransactionPrice(value)
+})
+
+interface Item {
+  label: string
+  value: string
+  tipMessage?: string
+}
+
+const items = computed((): Item[] => [
+  {
+    label: `${t('redeem_holofuel.request_id')}:`,
+    value: modalProps.value.requestId
+  },
+  {
+    label: `${t('$.date')}:`,
+    value: formattedDate.value
+  },
+  {
+    label: `${t('redeem_holofuel.holo_fuel_amount')}:`,
+    value: `${formattedHfAmount.value} HF`
+  },
+  {
+    label: `${t('redeem_holofuel.redemption_currency')}:`,
+    value: modalProps.value.currency
+  },
+  {
+    label: `${t('redeem_holofuel.redemption_amount')}:`,
+    value: `${formattedRedemptionAmount.value} HOT`,
+    tipMessage: t('redemption_history.transaction_price', { hf: 1, hot: 1 })
```

Comment:
It appears there are now two references to `redemption_history`. Can we collapse one into the other as  a `header` or `title`?  If not, we should rename one.

Reply:
...also why does `transaction_price` live on `redemption_history`?  It will be something that is needed and accessible *prior* to making a request to `redeem` holofuel, and therefore isn't necessarily related to the `redemption_history`.